src/south/FolderScanner/FolderScanner.class.js

```diff
@@ -25,6 +25,10 @@ class FolderScanner extends ProtocolHandler {
     this.minAge = minAge
     this.regex = new RegExp(regex)
     this.compression = compression
+
+    this.lastOnScanAt = null
+    this.lastAddFileAt = null
```

Comment:
this.lastOnScanAt = null
    this.lastAddFileAt = null
could be part of the ProtocolHandlerClass